controllers/articles.controller.js

```diff
@@ -51,11 +50,32 @@ function postComByArticle_id(request,response,next){
     })
     .catch((error)=>{
            next(error)
+    })  
+}
+
+function patchVotesByArticle_id(request,response,next){
+    const {article_id} = request.params
+    const {inc_votes} = request.body
+    selectArticleById(article_id)
+    .then((result)=>{
+        if (!inc_votes || typeof inc_votes !== 'number' ){ //request body missing || invalid vote type
```

Comment:
checking the inc_votes could probably be handled by the database and a PSQL error message.